x/twap/api_test.go

```diff
@@ -8,7 +8,36 @@ import (
 	"github.com/osmosis-labs/osmosis/v10/x/twap/types"
 )
 
-var ThreePlusOneThird sdk.Dec = sdk.MustNewDecFromStr("3.333333333333333333")
+type getTwapInput struct {
+	poolId          uint64
+	quoteAssetDenom string
+	baseAssetDenom  string
+	startTime       time.Time
+	endTime         time.Time
+}
```

Comment:
If we're moving this struct up, we should remove the definitions in function, and the function to go from inputs to this